moveos/moveos-stdlib/moveos-stdlib/sources/context.move

```diff
@@ -1,7 +1,9 @@
+// Copyright (c) RoochNetwork
```

Comment:
Maybe can consider updating the framework license checking and adding logic to `scripts/update_licence.sh`, to ensure that subsequent frameworks have licenses.

Reply:
The `update_licence` skip the move file #386. 

If we add the license to the Move code, we can enable the check.

Reply:
The `update_licence` skip Move files #386. If we decide to add the license to Move files, we can enable it.